test/lib/waitForConnector.ts

```diff
@@ -1,10 +1,10 @@
 import { sleep } from "@js-soft/ts-utils";
 import axios from "axios";
-import { getTimeout } from "./setTimeout";
+import { getTimeout as getRetryCount } from "./setTimeout";
 
 export default async function waitForConnector(port: number): Promise<void> {
-    const maxRetries = 10;
-    const timeout = getTimeout(1000);
+    const maxRetries = getRetryCount(10);
+    const timeout = 1000;
```

Comment:
was this accidentally added?

Reply:
nop needed to move it from timeout to retires because when timeout is set to 99999 and tests are running fine the first check takes forever

Reply:
IC